servers/keycloak_deployed_mcp.py

```diff
@@ -0,0 +1,213 @@
+import logging
+import os
+import uuid
+from datetime import date
+from enum import Enum
+from typing import Annotated
+
+from azure.cosmos.aio import CosmosClient
+from azure.identity.aio import DefaultAzureCredential, ManagedIdentityCredential
+from dotenv import load_dotenv
+from fastmcp import FastMCP
+from fastmcp.server.auth import RemoteAuthProvider
+from fastmcp.server.auth.providers.jwt import JWTVerifier
+from pydantic import AnyHttpUrl
+
+load_dotenv(override=True)
+
+logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(message)s")
+logger = logging.getLogger("ExpensesMCP")
+
+
+def require_env_var(name: str) -> str:
```

Comment:
I recommend just using os.environ[] instead of a wrapper function, it's less code and will still exit(1) when its not defined.

Reply:
consolidated into the other one, so this is resolved there.